README.md

```diff
@@ -46,7 +46,7 @@ Locksmith needs to run on a domain joined system.
 
 ## Script
 ### Download the standalone script (classic) without module:
-1. Download the latest script version: [https://github.com/TrimarcJake/Locksmith/releases/latest/download/Invoke-Locksmith.ps1](https://github.com/TrimarcJake/Locksmith/releases/latest/download/Invoke-Locksmith.ps1)
+1. Download the latest script version: [https://github.com/TrimarcJake/Locksmith/releases/latest/download/Invoke-Locksmith.ps1](https://github.com/TrimarcJake/Locksmith/releases/latest/download/Invoke-Locksmith.zip)
```

Comment:
If changing URL target to a .zip file, the display text also needs to be changed from .ps1 to .zip. 👍